### PR TITLE
refactor(rxUnsafeRemoteHTML): FRMW-403-convert to quark filter

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-403

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxPageTitle.rxUnsafeRemoteHTML` to Quark filter.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `rxUnsafeRemoteHTML` Quark Filter

### Demo Page
screen shot 2015-12-22 at 3 35 17 pm

### API Docs
screen shot 2015-12-22 at 3 35 47 pm

**Comment by glynnis** [comment]
Design LGTM

File: src/components/rxPageTitle/rxPageTitle.module.js [hunk 1]
```diff
@@ -6,11 +6,9 @@
...
- *
- * ## Filters
- * * {@link rxPageTitle.filter:rxUnsafeRemoveHTML rxUnsafeRemoveHTML}
- *
  * ## Services
  * * {@link rxPageTitle.service:rxPageTitle rxPageTitle}
  */
-angular.module('encore.ui.rxPageTitle', []);
+angular.module('encore.ui.rxPageTitle', [
+	'encore.ui.quarks'
```
Comment (CITguy) [inline src/components/rxPageTitle/rxPageTitle.module.js]: please reduce to one indentation level (4 spaces)

File: src/quarks/rxUnsafeRemoveHTML/rxUnsafeRemoveHTML.meta.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "displayName": "rxUnsafeRemoveHTML",
+    "stability": "stable",
+    "description": "Removes all HTML tags a string.",
```
Comment (CITguy) [inline src/quarks/rxUnsafeRemoveHTML/rxUnsafeRemoveHTML.meta.json]: (grammar) "Removes all HTML tags **from** a string."

File: src/quarks/rxUnsafeRemoveHTML/scripts/rxUnsafeRemoveHTML.spec.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ describe('rxUnsafeRemoveHTML', function () {
     var unsafeRemove;
 
     beforeEach(function () {
-        module('encore.ui.rxPageTitle');
+       module('encore.ui.quarks');
```
Comment (CITguy) [inline src/quarks/rxUnsafeRemoveHTML/scripts/rxUnsafeRemoveHTML.spec.js]: correct indentation (should be 8 leading spaces)

**Comment by 100stacks** [comment]
@CITguy comments implemented :+1: 

Also, I updated the **Demo Page** in https://github.com/rackerlabs/encore-ui/pull/1470#issuecomment-166742470 to provide more explanation to the user.

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**